portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -141,3 +140,28 @@ async function fetchSubmittedLocations() {
 
   return commentData;
 }
+
+google.charts.load('current', {'packages': ['corechart']});
+google.charts.setOnLoadCallback(drawStateIncarcerationChart);
+
+/** Creates chart and adds it to the page. */
+function drawStateIncarcerationChart() {
```

Comment:
Non-coding comment: Mentioning the source of the data somewhere is important for credibility.

Reply:
We planned to have a sources section on the statistics page where we include how where we sourced our data from.